.github/workflows/dev-pipeline.yml

```diff
@@ -17,7 +17,7 @@ env:
   OPTIONS: "--region=europe-west2 --memory=256MB --allow-unauthenticated"
   SHEET_ID: ${{ secrets.DEV_SHEET_ID }}
   COMMUNITY_SHEET_ID: ${{ secrets.DEV_COMMUNITY_SHEET_ID }}
-  WORKSHEET_NAME: ${{ secrets.DEV_WORKSHEET_NAME }}
+  WORKSHEET_NAME: ${{ secrets.TRIAL_WORKSHEET_NAME }}
```

Comment:
Should this be an additional ENV VAR if we need two spreadsheets? If we just need to point to a replacement one we can update the secret to use the new sheet ID.